src/components/comparison/comparison-side/comparison-side.tsx

```diff
@@ -208,7 +214,11 @@ export const ComparisonSide = ({
       setActiveButton(ActiveButton.memory);
       setLoadNumber((prev) => prev + 1);
     }
-  }, [compareButtonMode]);
+
+    if (compareButtonMode === CompareButtonMode.Start && loadingStopped) {
+      setActiveButton(ActiveButton.options);
+    }
```

Comment:
I don't like this approach. You should not swith to stats panel from the start. If comparison was finished successfully you shoud switch to stats at this moment.